CONTRIBUTING.md

```diff
@@ -20,19 +20,19 @@ This document covers contributions to this git repository.
 ## Open source
 
 While this repository does not contain open source code, we manage content contributions following open source practice, as detailed below.
-All contributions to this project will be released under open source license as described in [LICENSE.md](/LICENSE.md).
+All contributions to this project will be released under open source license as described in [LICENSE](/LICENSE).
 By submitting a pull request (PR), you are agreeing to release the PR contents under this license.
 
 ## Communication
 
-Anyone interested in contributing should join the mailing list and other [communication channels](https://github.com/cncf/wg-env-sustainability#meetings)
+Anyone interested in contributing should join the mailing list and other [communication channels](https://github.com/cncf/wg-env-sustainability#contact)
 
 We strongly encourage and support all our members to participate in anyway they can.
 Not everyone can participate in the regularly scheduled live meetings, so we strive to make our processes friendly for people to be active contributors through asynchronous communication and contributions to our documentation in this repository.
 
 ## Github pull requests and issues
 
-If you are new to the group, [reviewing pull requests](pull-request-review.md) and commenting on issues is a great way to get involved!
+If you are new to the group, [reviewing pull requests](https://github.com/dili91/tag-env-sustainability/pulls) and commenting on issues is a great way to get involved!
 
 When creating or reviewing pull requests, please refer to the [writing style guide](writing-style.md) to help maintain consistency across all of our documents.
```

Comment:
I could not find a`writing style`. Happy to track this with an issue if it helps